src/github-reader.ts

```diff
@@ -13,7 +13,7 @@
 
 import { ProjectReader, PathKind } from './deploy-struct'
 import { GithubDef, makeClient, readContents, seemsToBeProject, OctokitNonArrayResponse } from './github'
-import * as Octokit from '@octokit/rest'
+import type { Octokit } from '@octokit/rest'
```

Comment:
Did we previously have an issue about using `import type` because it's too new a feature and wasn't supported in some runtime we were using - or am I imagining this? :)

Reply:
Yes, it did cause problem due to the use of pre 3.8 version in the workbench, thanks for catching this!